api-specificatie/openapi.yaml

```diff
@@ -0,0 +1,291 @@
+        '410':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/410"
+        '415':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/415"
+        '429':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/429"
+        '500':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/500"
+        '503':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/503"
+      tags:
+      - gewijzigdepersonen
+components:
+  schemas:
+    VolgindicatiePersonenHalCollectie:
+      type: object
+      properties:
+        _links:
+          $ref: "#/components/schemas/VolgindicatiePersonenHalCollectionLinks"
+        burgerservicenummers:
+          type: array
+          items:
+            $ref: '#/components/schemas/VolgindicatiePersonenHal'
+    VolgindicatiePersonenHalCollectionLinks:
+      type: object
+      properties:
+        self:
+          $ref: "#/components/schemas/VolgindicatiePersonenHalLink"
+    VolgindicatiePersonenHal:
+      allOf:
+        - $ref: '#/components/schemas/Burgerservicenummer'
+    VolgindicatiePersonenHalLink:
+      description: De Link Object zoals gespecificeerd in https://tools.ietf.org/html/draft-kelly-json-hal-08#section-5
+      type: object
+      required:
+        - href
+      properties:
+        href:
+          $ref: "#/components/schemas/VolgindicatiePersonenHref"
+        title:
+         description: "Voor mens leesbaar label bij de link"
+         type: string
+    VolgindicatiePersonenHref:
+      type: string
+      example: "https://datapunt.voorbeeldgemeente.nl/api/v{major-versie}/volgindicatiepersonen"
+    VolgindicatiePersonen:
+      required:
+        - burgerservicenummers
+      type: object
+      properties:
+        burgerservicenummers:
+          type: array
+          items:
+            $ref: '#/components/schemas/Burgerservicenummer'
+    GewijzigdepersonenHalCollectie:
+      type: object
+      properties:
+        _links:
+          $ref: "#/components/schemas/GewijzigdepersonenHalCollectionLinks"
+        burgerservicenummers:
+          type: array
+          items:
+            $ref: '#/components/schemas/GewijzigdepersonenHal'
+    GewijzigdepersonenHalCollectionLinks:
+      type: object
+      properties:
+        self:
+          $ref: "#/components/schemas/GewijzigdepersonenHalLink"
+        ingeschrevenPersoon:
+          $ref: "#/components/schemas/Gewijzigdepersoon_Link"
+    GewijzigdepersonenHal:
```

Comment:
hier is denk ik niks Hal aan, toch? Is alleen burgerservicenummer, dus type string

Reply:
Volgens mij heb je in deze API alleen Hal nodig voor de paginering en de collecties. Opmerking van Frank gaat ook op voor VolgindicatiePersonenHal

Reply:
Dit component is n.a.v. andere aanpassingen komen te vervallen.